index.html

```diff
@@ -1728,10 +1728,10 @@ <h2>Multihash</h2>
 
           <p>
 A Multihash value starts with a binary header, which includes 1) an identifier
-for the specific cryptographic hash algorithm, 2) the length of the
-cryptographic hash, and 3) the value of the cryptographic hash. The normative
-Multihash header values defined by this specification, and their associated
-output sizes and associated specifications, are provided below:
+for the specific cryptographic hashing algorithm, 2) a cryptographic digest size
```

Comment:
Line 1792 changed from `hashing` to `hash`, while line 1731 changed from `hash` to `hashing`. These lines should use the same word. I think `hashing` is better in this case, but I would accept `hash`.

Similarly, line 1793 uses `cryptographic digest length` while line 1731 is being changed from `length of the cryptographic hash` to `cryptographic digest size`. Again, these should be consistent. I think `length of the cryptographic digest` is better, with a fallback of `cryptographic digest length`.

Reply:
I *think* I got all instances, but I can't tell. A previous rebase threw off the line numbers. Please check to make sure I did what you wanted @TallTed ... because I agree with your suggestions/preferences.